runtime/compiler/control/JitDump.cpp

```diff
@@ -51,13 +51,13 @@ struct ILOfCrashedThreadParamenters
    : vmThread(vmThread), comp(comp), jitdumpFile(jitdumpFile)
       {}
 
-   /// The JVM thread backing the crashed thread
+   /** The JVM thread backing the crashed thread*/
    J9VMThread *vmThread;
 
-   /// The compilation object extracted from the crashed thread
+   /** The compilation object extracted from the crashed thread*/
    TR::Compilation *comp;
 
-   /// The jitdump file to trace the IL of the crashed thread to
+   /** The jitdump file to trace the IL of the crashed thread to*/
```

Comment:
I think it's easier to spot the `*/` if it appears near the beginning of the line rather than the end of a line.  May I ask you to change these to have this format?

```
   /** The JVM thread backing the crashed thread
    */
```